specification/iotcentral/data-plane/Microsoft.IoTCentral/stable/1.0.0/iotcentral.json

```diff
@@ -0,0 +1,2262 @@
+        "value": {
+          "description": "The collection of users.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/User"
+          }
+        },
+        "nextLink": {
+          "description": "URL to get the next page of users.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "X509": {
+      "type": "object",
+      "properties": {
+        "clientCertificates": {
+          "description": "The X.509 client certificates for this credential.",
+          "$ref": "#/definitions/X509Certificates"
+        }
+      }
+    },
+    "X509Attestation": {
+      "x-ms-discriminator-value": "x509",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Attestation"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "x509": {
+              "description": "The X.509 credentials for this attestation.",
+              "$ref": "#/definitions/X509"
+            }
+          },
+          "required": [
+            "x509"
+          ]
+        }
+      ]
+    },
+    "X509Certificate": {
+      "type": "object",
+      "properties": {
+        "certificate": {
+          "description": "The string representation of this certificate.",
+          "type": "string"
+        },
+        "info": {
+          "description": "Information about this certificate.",
+          "readOnly": true,
+          "$ref": "#/definitions/X509CertificateInfo"
+        }
+      }
+    },
+    "X509CertificateInfo": {
+      "type": "object",
+      "properties": {
+        "sha1Thumbprint": {
+          "description": "The SHA-1 hash value of the certificate.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "sha1Thumbprint"
+      ]
+    },
+    "X509Certificates": {
+      "type": "object",
+      "properties": {
+        "primary": {
+          "description": "The primary X.509 certificate for this credential.",
+          "$ref": "#/definitions/X509Certificate"
+        },
+        "secondary": {
+          "description": "The secondary X.509 certificate for this credential.",
+          "$ref": "#/definitions/X509Certificate"
+        }
+      },
+      "required": [
+        "primary"
+      ]
+    }
+  },
+  "paths": {
+    "/apiTokens": {
+      "get": {
+        "operationId": "ApiTokens_List",
+        "summary": "Get the list of API tokens in an application. The token value will never be returned for security reasons.",
+        "x-ms-examples": {
+          "List API tokens": {
+            "$ref": "./examples/apitoken_list.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ApiTokenCollection"
+            }
+          }
+        }
+      }
+    },
+    "/apiTokens/{token_id}": {
```

Comment:
Is this the standard variable style for swagger? I wonder how this will generate with autorest for different languages. I'm used to seeing camelCase like here in the [ADT swagger](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/digitaltwins/data-plane/Microsoft.DigitalTwins/stable/2020-10-31/digitaltwins.json#L523).

Reply:
Fixed to use same pattern as ADT.